packages/mux-audio-react/src/index.tsx

```diff
@@ -33,6 +36,7 @@ const MuxAudio = React.forwardRef<HTMLAudioElement | undefined, Partial<Props>>(
       startTime,
       src: outerSrc,
       children,
+      autoPlay,
```

Comment:
this removes `autoPlay` from `restProps`, so, it doesn't get added to the `<audio>` element itself.